src/frontend/packages/lib_common/src/queries/actionOne.spec.tsx

```diff
@@ -20,8 +20,8 @@ describe('queries/actionOne', () => {
       object: objectToUpdate,
```

Comment:
You shouldn't have to change this file. If eslint is unhappy you should disable rules for tests.

Reply:
I am not sure, why are we allowed to disable esrules in test?

Reply:
Because its handy to assert or use any in tests, since the main issue we could face is the code to fail and throw, and that's not an issue in a test because this test will fail.